demo/src/examples/6 Touch-Friendly Features.vue

```diff
@@ -66,6 +66,9 @@
     }
   });
 
+  // `maxTouchPoints` isn't recognized by TS. Safe to ignore.
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Are both ts-ignore and eslint-ignore needed?

Reply:
@wipfli yes. `ts-ignore` is prohibited by the ESList :smile: ESLint ignore is allowed at any place.